### DISCUSSION TITLE
[Question]: GPT-3 token usage w/o me using it

### DISCUSSION THREAD

**Discussion by darkBuddha** [danny-avila/LibreChat#1630] [Q&A] [answered]
### What is your question?

Hi, I noticed regular (little) activity to GPT-3 when I use LibreChat, but it is disabled in my `.env`.

Is it possible that LibreChat connects on its own to detect API availability, or something similar? It amounts to ~$0.30 per month. Should I delete old chats that were connected to LibreChat? I suspected maybe plugins, but it happens also when they are fully disabled.

I switched API keys to rule out that something else is creating the API accesses. 

Any idea how to investigate further?

### More Details

`GPT-3.5-turbo-0613
`
### Screenshots

![Screenshot 2024-01-25 at 09 35 04](https://github.com/danny-avila/LibreChat/assets/3326929/1abc2fe2-a4f4-4b72-86b0-fc50aab5eca5)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
It is from conversation titles. You can disable it with this env variable:

TITLE_CONVO=false

**Reply by darkBuddha** [reply]
This makes sense! Thank you for taking the time to respond.
